src/webinars.njk

```diff
@@ -130,12 +134,19 @@ meta:
                     <a href="{{ item.url }}" class="w-full flex flex-col group hover:no-underline">
                         <div class="">
                             <time class="block text-xs mb-2" value="{{ item.date }}" class="text-gray-400">{{ item.date | shortDate }}</time>
-                            <div>{% if item.image %}<img class="w-full h-auto" src="{{item.image}}" />{% else %}{{ item.url | generatePostSVG |safe}}{% endif %}</div>
```

Comment:
`item.image` appears to be a bug

Reply:
Why a bug? Isn't that intended to add an image if the front matter defines one?

Reply:
this is fixed in https://github.com/flowforge/website/pull/460

Reply:
> Why a bug? Isn't that intended to add an image if the front matter defines one?

it used to work, but when we upgraded to Eleventy 1.0, I think I missed this one as we didn't have any past webinars in that flagged it.